types/packageensure.pp

```diff
@@ -0,0 +1,6 @@
+type Simplib::PackageEnsure = Enum[
+  'latest',
+  'absent',
+  'present',
+  'installed'
```

Comment:
Needs `purged` and `held`.

https://docs.puppet.com/puppet/latest/type.html#package-attribute-ensure